kivy/parser.py

```diff
@@ -150,5 +150,13 @@ def parse_float4(text):
     return value
 
 
-parse_int = int
-parse_float = float
+def parse_int(text):
```

Comment:
In that case the existing doc display is ok. You are adding a extra method call where it is really not needed. Just for the sake of keeping doc uniform.